packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -752,13 +752,8 @@ class PageTransitionsTheme with Diagnosticable {
     Animation<double> animation,
     Animation<double> secondaryAnimation,
     Widget child,
+    TargetPlatform platform,
```

Comment:
This is effectively a breaking change for both callers and subclassers (it does break existing internal Google apps).  On the upside, it does not breaking any existing Flutter repo or customer tests (flutter/tests). A brief migration will be needed per https://docs.flutter.dev/resources/compatibility. One of the reviewers can update the internal Google apps.

Reply:
Should I create a migration guide in the `website` repository look like flutter/website#8976?

Reply:
https://github.com/flutter/website/pull/8976 looks like it covers the changes. Will land this PR and make the internal updates.